test/functional/apps/dashboard/_embeddable_rendering.js

```diff
@@ -32,28 +32,39 @@ export default function ({ getService, getPageObjects }) {
   const browser = getService('browser');
   const dashboardExpect = getService('dashboardExpect');
   const dashboardAddPanel = getService('dashboardAddPanel');
+  const renderable = getService('renderable');
   const PageObjects = getPageObjects(['common', 'dashboard', 'header', 'visualize', 'discover']);
 
   const expectAllDataRenders = async () => {
     await dashboardExpect.pieSliceCount(16);
-    await dashboardExpect.seriesElementCount(19);
-    await dashboardExpect.dataTableRowCount(5);
-    await dashboardExpect.savedSearchRowCount(50);
-    await dashboardExpect.goalAndGuageLabelsExist(['63%', '56%', '11.915 GB']);
-    await dashboardExpect.inputControlItemCount(5);
     await dashboardExpect.metricValuesExist(['7,544']);
-    await dashboardExpect.markdownWithValuesExists(['I\'m a markdown!']);
-    await dashboardExpect.lineChartPointsCount(5);
-    await dashboardExpect.tagCloudWithValuesFound(['CN', 'IN', 'US', 'BR', 'ID']);
-    await dashboardExpect.timelionLegendCount(0);
+    await dashboardExpect.seriesElementCount(19);
     const tsvbGuageExists = await find.existsByCssSelector('.tvbVisHalfGauge');
     expect(tsvbGuageExists).to.be(true);
-    await dashboardExpect.tsvbMetricValuesExist(['210,007,889,606']);
-    await dashboardExpect.tsvbMarkdownWithValuesExists(['Hi Avg last bytes: 6286.674715909091']);
-    await dashboardExpect.tsvbTableCellCount(20);
+    await dashboardExpect.timelionLegendCount(0);
+    await dashboardExpect.markdownWithValuesExists(['I\'m a markdown!']);
+    await dashboardExpect.vegaTextsExist(['5,000']);
+    await dashboardExpect.goalAndGuageLabelsExist(['63%', '56%', '11.915 GB']);
+    await dashboardExpect.dataTableRowCount(5);
+    // missing bar chart
+    await dashboardExpect.tagCloudWithValuesFound(['CN', 'IN', 'US', 'BR', 'ID']);
+    // missing region map
+    // missing tsvb gauge
     await dashboardExpect.tsvbTimeSeriesLegendCount(1);
+    // missing geo map
+    // sum the two input control embeddables
+    await dashboardExpect.inputControlItemCount(5);
+    await dashboardExpect.tsvbTableCellCount(20);
+    await dashboardExpect.tsvbMarkdownWithValuesExists(['Hi Avg last bytes: 6286.674715909091']);
     await dashboardExpect.tsvbTopNValuesExist(['5,734.79', '6,286.675']);
-    await dashboardExpect.vegaTextsExist(['5,000']);
+    await dashboardExpect.tsvbMetricValuesExist(['210,007,889,606']);
+    // missing animal sound pie
+    // sum area chart + non timebased line chart points
+    await dashboardExpect.lineChartPointsCount(5);
+    // missing scripted filter and query
```

Comment:
Yes, we have 28 visualization on that dashboard, but we tested only some of them. I've just add that comment to remember the type and the order of the missing visualization. I can add an issue to remember to add the missing tests on a different PR.

Reply:
Gotcha.  How about putting all the missing ones in one block?  I think that'll make it a little clearer and then can write out more of a statement so it's a little more obvious.  e.g. "These types of charts aren't tested with any of the above checks: ...".  Could you also write what kind of chart it is?  I forget what "scripted filter and query" one is for instance (if it's easy for you to remember, if not, don't worry about it).  Easier to track which kinds of visualizations are missing the tests for, since the checks sum a bunch of stuff up.  Which, is def more difficult to follow.  I like the descriptions you added, makes it easier to follow!

I thought I had covered most of them, so surprised so many are missing.